catalog/security/banner-marking/src/test/groovy/org/codice/alliance/security/banner/marking/BannerCommonMarkingExtractorSpec.groovy

```diff
@@ -319,6 +312,114 @@ class BannerCommonMarkingExtractorSpec extends Specification {
+        try {
+            extractor.processReleasability(metacard, bannerMarkings)
+        } catch (RuntimeException rte) {
+            validationException = rte
+        }
+
+        then:
+        validationException != null
+        validationException.getMessage() == "Extracted security attribute security.releasability" +
+                " from the banner markings does not match value pre-existing on the" +
+                " metacard. The metacard will not be created."
+    }
+
+    def 'test security conflict codewords'() {
+        when:
+        initBannerMarkings("SECRET//TK//REL TO CAN, DEU, NZL")
+        metacard.setAttribute(new AttributeImpl(Security.CODEWORDS, "CONFLICTING VALUE"))
+        validationException = null
+        try {
+            extractor.processCodewords(metacard, bannerMarkings)
+        } catch (RuntimeException rte) {
+            validationException = rte
+        }
+
+        then:
+        validationException != null
+        validationException.getMessage() == "Extracted security attribute security.codewords" +
+                " from the banner markings does not match value pre-existing on the" +
+                " metacard. The metacard will not be created."
+    }
+
+    def 'test security conflict dissemination'() {
+        when:
+        initBannerMarkings("SECRET//NOFORN/PROPIN")
+        metacard.setAttribute(new AttributeImpl(Security.DISSEMINATION_CONTROLS, "CONFLICTING VALUE"))
+        validationException = null
+        try {
+            extractor.processDissem(metacard, bannerMarkings)
+        } catch (RuntimeException rte) {
+            validationException = rte
+        }
+
+        then:
+        validationException != null
+        validationException.getMessage() == "Extracted security attribute security.dissemination-controls" +
+                " from the banner markings does not match value pre-existing on the" +
+                " metacard. The metacard will not be created."
+    }
+
+    def 'test security conflict owner producer'() {
+        when:
+        initBannerMarkings("//COSMIC TOP SECRET//BALK")
+        metacard.setAttribute(new AttributeImpl(Security.OWNER_PRODUCER, "CONFLICTING VALUE"))
+        validationException = null
+        try {
+            extractor.processOwnerProducer(metacard, bannerMarkings)
+        } catch (RuntimeException rte) {
+            validationException = rte
+        }
+
+        then:
+        validationException != null
+        validationException.getMessage() == "Extracted security attribute security.owner-producer" +
+                " from the banner markings does not match value pre-existing on the" +
+                " metacard. The metacard will not be created."
+    }
+
+    def 'test security conflict classification system'() {
+        when:
+        initBannerMarkings("//COSMIC TOP SECRET//BALK")
+        metacard.setAttribute(new AttributeImpl(Security.CLASSIFICATION_SYSTEM, "CONFLICTING VALUE"))
```

Comment:
✏️ I think your tests would be more definitive and less prone to being broken in the future if you used real values. So instead of `CONFLICTING VALUE` here, use a real class system value such as `USA`